### PR TITLE
Add support for requesting organizations, courses and course details

### PR DESCRIPTION
**Pull request by jesseanttila-cai**
Solves #46

### CODE REVIEW

**Comment by jolampi** [comment]
There was a merge conflict in `package.json`

**Comment by jolampi** [comment]
> There was a merge conflict in `package.json`
And the conflict editor displayed tabs as single spaces...

File: src/api/tmc.ts [hunk 1]
```diff
@@ -65,4 +67,123 @@ export default class TMC {
...
+    attempted: boolean;
+    completed: boolean;
+    reviewed: boolean;
+    all_review_points_given: boolean;
+    memory_limit: number | null;
+    runtime_params: string[];
+    valgrind_strategy: string;
+    code_review_requests_enabled: boolean;
+    run_tests_locally_action_enabled: boolean;
+};
```
Comment (HegePI) [inline src/api/tmc.ts]: `type Exercise = {
    id: number;
    name: string;
    locked: boolean;
    deadline_description: string | null;
    deadline: string | null;
    soft_deadline: string | null;
    soft_deadline_description: string | null;
    checksum: string;
    return_url: string;
    zip_url: string;
    returnable: boolean;
    requires_review: boolean;
    attempted: boolean;
    completed: boolean;
    reviewed: boolean;
    all_review_points_given: boolean;
    memory_limit: number | null;
    runtime_params: string[];
    valgrind_strategy: string;
    code_review_requests_enabled: boolean;
    run_tests_locally_action_enabled: boolean;
    exercise_submissions_url: string;
    latest_submission_url: string;
    latest_submission_id: number;
    solution_zip_url: string
};`

Comment (HegePI) [reply src/api/tmc.ts]: added missing fields to type Exercise.

**Pull request merged**